build/target/etc/pm2-lisk.json

```diff
@@ -0,0 +1,18 @@
+{
+        "apps": [{
+                "name": "lisk.app",
+                "script": "app.js",
```

Comment:
probably this should be changed to `src/index.js`

Reply:
Correct but this will be done in #3041